### PR TITLE
Add zigbee2mqtt template

### PR DESCRIPTION
**Pull request by mar005**

### CODE REVIEW

File: templates/zigbee2mqtt.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Zigbee2MQTT</Name>
+  <Repository>koenkk/zigbee2mqtt</Repository>
+  <Registry>https://hub.docker.com/r/koenkk/zigbee2mqtt/</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://www.zigbee2mqtt.io/information/docker.html</Support>
+  <Project/>
```
Comment (Roxedus) [inline templates/zigbee2mqtt.xml:9]: Please fill this

File: templates/zigbee2mqtt.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <Overview>Zigbee &#x1F41D; to MQTT bridge &#x1F309; using zigbee-shepherd &#x1F415;   Converted By Community Applications   Always verify this template (and values) against the dockerhub support page for the container</Overview>
+  <Category/>
```
Comment (Roxedus) [inline templates/zigbee2mqtt.xml:11]: Would be nice is this was filled
[link](https://selfhosters.net/docker/templating/templating/#21-categories)

File: templates/zigbee2mqtt.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <WebUI/>
```
Comment (Roxedus) [inline templates/zigbee2mqtt.xml:12]: Remove unnecessary fields

File: templates/zigbee2mqtt.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Zigbee2MQTT</Name>
+  <Repository>koenkk/zigbee2mqtt</Repository>
+  <Registry>https://hub.docker.com/r/koenkk/zigbee2mqtt/</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://www.zigbee2mqtt.io/information/docker.html</Support>
+  <Project/>
+  <Overview>Zigbee &#x1F41D; to MQTT bridge &#x1F309; using zigbee-shepherd &#x1F415;   Converted By Community Applications   Always verify this template (and values) against the dockerhub support page for the container</Overview>
```
Comment (Roxedus) [inline templates/zigbee2mqtt.xml:10]: This will be rejected from CA. Alter this text

File: templates/zigbee2mqtt.xml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  <Category/>
+  <WebUI/>
+  <Icon>https://raw.githubusercontent.com/mar005/unRAID-CA-templates/master/templates/img/zigbee2mqtt.png</Icon>
+  <Description>Zigbee &#x1F41D; to MQTT bridge &#x1F309; using zigbee-shepherd &#x1F415;   Converted By Community Applications   Always verify this template (and values) against the dockerhub support page for the container</Description>
+  <Config Name="Config Directory" Target="/app/data" Default="/mnt/user/appdata/zigbee2mqtt/" Mode="rw" Description="Container Path: /app/data" Type="Path" Display="always" Required="true" Mask="false"/>
+  <Config Name="Zigbee module" Target="" Default="PATH_TO_ZIGBEE_MODULE" Mode="" Description="Path to zigbee module, preferably serial/by-id" Type="Device" Display="always" Required="true" Mask="false"/>
+  <Config Name="Timezone" Target="TZ" Default="America/Chicago" Mode="" Description="Enter your timezone" Type="Variable" Display="always" Required="true" Mask="false"/>
```
Comment (Roxedus) [inline templates/zigbee2mqtt.xml:17]: Redundant. Automatically set by unraid

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: templates/zigbee2mqtt.xml (hunk 1 above)
Comment (mar005) [reply templates/zigbee2mqtt.xml:9]: Sorry fill what specifically?

Comment (Roxedus) [reply templates/zigbee2mqtt.xml:9]: The link to the project

File: templates/zigbee2mqtt.xml (hunk 5 above)
Comment (mar005) [reply templates/zigbee2mqtt.xml:16]: Did you mean the timezone? Will a variable appear automatically by unraid?

**Review by mar005 — COMMENTED** [review]
Made changes

File: templates/zigbee2mqtt.xml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Zigbee2MQTT</Name>
+  <Repository>koenkk/zigbee2mqtt</Repository>
+  <Registry>https://hub.docker.com/r/koenkk/zigbee2mqtt/</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://www.zigbee2mqtt.io/information/docker.html</Support>
+  <Project>https://www.zigbee2mqtt.io/</Project>
+  <Overview>Zigbee to MQTT bridge using zigbee-shepherd allows you to use your Zigbee devices without the vendors bridge or gateway. It bridges events and allows you to control your Zigbee devices via MQTT. In this way you can integrate your Zigbee devices with whatever smart home infrastructure you are using. Converted By Community Applications   Always verify this template (and values) against the dockerhub support page for the container</Overview>
```
Comment (Roxedus) [inline templates/zigbee2mqtt.xml:10]: This will probably still be rejected by CA

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: templates/zigbee2mqtt.xml (hunk 5 above)
Comment (Roxedus) [reply templates/zigbee2mqtt.xml]: yes

File: templates/zigbee2mqtt.xml (hunk 6 above)
Comment (mar005) [reply templates/zigbee2mqtt.xml:10]: Hey @Roxedus mind being more specific on which part makes it rejected? I changed the overview to not have characters that are html encoded, but now i do not understand what is missing or needs to be removed.

Comment (Roxedus) [reply templates/zigbee2mqtt.xml:10]: The fact that it says converted from CA, and everything related

**Review by mar005 — COMMENTED** [review]
Updated overview and description

**Review by Roxedus — APPROVED** [review]

**Pull request merged**